infrastructure/main.tf

```diff
@@ -0,0 +1,54 @@
+provider "azurerm" {}
+
+provider "vault" {
+  address = "https://vault.reform.hmcts.net:6200"
+}
+
+locals {
+  app = "bulk-scanning"
+}
+
+module "backend" {
+  source       = "git@github.com:hmcts/moj-module-webapp?ref=master"
+  product      = "${var.product}-${local.app}"
+  location     = "${var.location}"
+  env          = "${var.env}"
+  ilbIp        = "${var.ilbIp}"
+  subscription = "${var.subscription}"
+  is_frontend  = false
+  capacity     = "${var.capacity}"
+
+  asp_name = "bulk-scanning"
```

Comment:
asp_name?

Reply:
Yes, copied over from rhubarb service. The variable then is used as part of the `asp_name` in deployment template.

Reply:
it's not required.asp_name is the name of the shared service plan to be deployed to. Name should follow ${product}-${env}-asp format